chapter_appendix_math/statistics_vn.md

```diff
@@ -17,38 +17,38 @@ To be able to discuss uncertainty in estimated values, we must learn some statis
 
 
 <!--
-The earliest reference of *statistics* can be traced back to an Arab scholar Al-Kindi in the $9^{\mathrm{th}}$-century, who gave a detailed description of how to use statistics and frequency analysis to decipher encrypted messages. 
-After 800 years, the modern statistics arose from Germany in 1700s, when the researchers focused on the demographic and economic data collection and analysis. 
-Today, statistics is the science subject that concerns the collection, processing, analysis, interpretation and visualization of data. 
+The earliest reference of *statistics* can be traced back to an Arab scholar Al-Kindi in the $9^{\mathrm{th}}$-century, who gave a detailed description of how to use statistics and frequency analysis to decipher encrypted messages.
```

Comment:
@quangnhat185 nếu em muốn xóa trailing space cho toàn bộ file thì có thể tạo một PR riêng cho việc đó. Nhưng việc này nên làm ở cuối sau khi toàn bộ các phần đã được dịch xong. Nếu không, rất dễ gây conflict.

Reply:
Ớ em đâu có đụng gì vào bản tiếng anh đâu sao nó lại báo có sự thay đổi ở đây nhỉ ?